OSDOCS-11837 OCP Release Notes 4.15.30 (#81177)
openshift/openshift-docs: release_notes/ocp-4-15-release-notes.adoc

For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
====

[id="ocp-4-15-30_{context}"]
=== RHSA-2024:6013 - {product-title} 4.15.30 bug fix and security update

Issued: 5 September 2024

{product-title} release 4.15.30, which includes security updates, is now available. The list of bug fixes that are included in this update is documented in the link:https://access.redhat.com/errata/RHSA-2024:6013[RHSA-2024:6013] advisory. The RPM packages that are included in this update are provided by the link:https://access.redhat.com/errata/RHSA-2024:6016[RHSA-2024:6016] advisory.

Space precluded documenting all of the container images for this release in the advisory.

You can view the container images in this release by running the following command:

[source,terminal]
----
$ oc adm release info 4.15.30 --pullspecs
----

[id="ocp-4-15-30-bug-fixes_{context}"]
==== Bug fixes

> * Previously, when running the `oc logs -f <pod>`command, the logs would not output anything after the log file was rotated. With this release, the kubelet outputs a log file after it has been rotated, and as a result the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-38861[*OCPBUGS-38861*])

Reviewer: ```suggestion
* Previously, when running the `oc logs -f <pod>` command, the logs would not output anything after the log file was rotated. With this release, the kubelet outputs a log file after it has been rotated, and as a result the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-38861[*OCPBUGS-38861*])
```

For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
====

[id="ocp-4-15-30_{context}"]
=== RHSA-2024:6013 - {product-title} 4.15.30 bug fix and security update

Issued: 4 September 2024

{product-title} release 4.15.30, which includes security updates, is now available. The list of bug fixes that are included in this update is documented in the link:https://access.redhat.com/errata/RHSA-2024:6013[RHSA-2024:6013] advisory. The RPM packages that are included in this update are provided by the link:https://access.redhat.com/errata/RHSA-2024:6016[RHSA-2024:6016] advisory.

Space precluded documenting all of the container images for this release in the advisory.

You can view the container images in this release by running the following command:

[source,terminal]
----
$ oc adm release info 4.15.30 --pullspecs
----

[id="ocp-4-15-30-enhancements_{context}"]
==== Enhancements

The following enhancement is included in this z-stream release:

[id="ocp-4-15-30-dockerfile-centos-9_{context}"]
===== CentOS update for Dockerfile

> * Previously, Dockerfile used CentOS Stream 8. With this release, Dockerfile now uses CentOS Stream 9. (link:https://issues.redhat.com/browse/OCPBUGS-39080[*OCPBUGS-39080*])

Author: Talked to the dev and he says this shouldn't get a release note after all.

[id="ocp-4-15-30-bug-fixes_{context}"]
==== Bug fixes

> * Previously, when running `oc logs -f <pod>`, the logs would not output anything after the log file was rotated. With this release, kubelet will output a log file after it has been rotated and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-38861[*OCPBUGS-38861*])

Reviewer: ```suggestion
* Previously, when running the `oc logs -f <pod>`command, the logs would not output anything after the log file was rotated. With this release, the kubelet outputs a log file after it has been rotated, and as a result the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-38861[*OCPBUGS-38861*])
```

> * Previously,an internal timeout would occur when the service account had short lived credentials. With this release, that timeout is removed and the timeout is now controlled by the parent context. (link:https://issues.redhat.com/browse/OCPBUGS-38198[*OCPBUGS-38198*])

Reviewer: ```suggestion
* Previously, an internal timeout would occur when the service account had short lived credentials. With this release, that timeout is removed, and the timeout is now controlled by the parent context. (link:https://issues.redhat.com/browse/OCPBUGS-38198[*OCPBUGS-38198*])
```
